test/styles/RRR.R

```diff
@@ -39,38 +48,110 @@ fun_call(argument1,
 fun_call(
     argument1,
     argument2
-    )
+)
 
 ## 3
 fun_call(parameter = (
     stuff
-    ),
-         argument)
+),
+argument)
 
 ## 4
 fun_call(parameter = fun_argument(
-             stuff
-    ),
-         argument)
+    argument1
+),
+argument2)
+
+## 5
+fun_call(parameter = fun_argument(argument1,
+                                  argument2
+                                  )
+        ,
+         argument3)
```

Comment:
Would it be nice if we had `ouuter-parameter` as follows:

If nil, no effect at all (other variables set the indent).

If `t`,

```
   some_function(parameter = other_function(
                                 argument
                             ))
 ```

If `(t)`:

```
   some_function(parameter = other_function(
                     argument
                 ))

```

And having it take precedence over `offset-argument` and `offset-argument-newline` unconditionally?